src/geometry/translation.rs

```diff
@@ -190,6 +192,40 @@ where DefaultAllocator: Allocator<N, D>
     }
 }
 
+impl<N: Scalar + ClosedAdd, D: DimName> Translation<N, D>
+where DefaultAllocator: Allocator<N, D>
```

Comment:
I did not add `transform_vector` or `inverse_transform_vector` inherent methods to `Translation`, because I thought they would be misleading; since they would be no-ops, calling them is likely to indicate user error. A user who needs geometric types to all consistently have a `transform_vector` capacity (such as for writing a macro) can still simply import the `Transformation` or `ProjectiveTransformation` traits.